simple_history/registry_tests/tests.py

```diff
@@ -88,28 +88,28 @@ class TestTrackingInheritance(TestCase):
 
     def test_tracked_abstract_base(self):
         self.assertEqual(
-            [
+            sorted([
```

Comment:
Why the `sorted` additions?

Reply:
Originally I added it because some of the tests were failing because the order of the meta fields changed when I supplied the `history_id`.  I just checked and that doesn't seem to be the case anymore maybe it was only the older versions of django that were supported at the time, but have since been dropped.  I can remove this change.

Reply:
Cool. Let's remove this